pyaedt/generic/design_types.py

```diff
@@ -823,8 +823,8 @@ def Maxwell3d(
     from pyaedt.maxwell import Maxwell3d as app
 
     return app(
-        projectname=projectname,
-        designname=designname,
+        project_name=projectname,
```

Comment:
if you change the init values for Maxwell you should do for all the app. I would suggest to keep this out from this PR and create a PR later to change the value for all the APP

Reply:
@maxcapodi78 why not start to change init values in the PR we are currently tackling?
You improve the class once and that's it.
We already have to open so many PRs for this refacto that I'd avoid to open others to change any other tachnicality in files that have already been refactored.